src/main/java/longtimenosee/logic/commands/PolicyDeleteAssignedCommand.java

```diff
@@ -22,8 +22,8 @@ public class PolicyDeleteAssignedCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": deletes assigned policy from a person. "
             + "Parameters: "
-            + "(Index of client) "
-            + "(Index of assigned policy) "
+            + "CLIENT_INDEX "
+            + "ASSIGNED_POLICY_INDEX "
             + "\nExample: " + COMMAND_WORD + " 1 1";
```

Comment:
![image](https://user-images.githubusercontent.com/97422591/199454423-8589320d-2c24-4335-81cf-e97bb864f8f2.png)

Likewise for parameters, newline should be added